src/components/NewDesign/Swap/Swap.tsx

```diff
@@ -0,0 +1,339 @@
+import React, { useEffect } from 'react'
+import { PublicKey } from '@solana/web3.js'
+import { BN } from '@project-serum/anchor'
+import { printBN, printBNtoBN } from '@consts/utils'
+import { Grid, Typography, Box, CardMedia } from '@material-ui/core'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import { OutlinedButton } from '@components/NewDesign/OutlinedButton/OutlinedButton'
+import ExchangeAmountInput from '@components/NewDesign/Inputs/ExchangeAmountInput/ExchangeAmountInput'
+import useStyles from './style'
+import { Status } from '@reducers/solanaWallet'
+import SwapArrows from '@static/svg/swap-arrows.svg'
+import { PRICE_DECIMAL } from '@consts/static'
+
+export interface SwapToken {
+  balance: BN
+  decimal: number
+  symbol: string
+  assetAddress: PublicKey
+}
+
+export interface Pools {
+  tokenX: {
+    _bn: PublicKey
```

Comment:
no, type of tokens adresses should be just PublicKey

Reply:
done